source/RaidFleet.h

```diff
@@ -0,0 +1,41 @@
+/* RaidFleet.h
+Copyright (c) 2023 by Hurleveur
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef RAID_FLEET_H_
+#define RAID_FLEET_H_
+
+#include <vector>
+
+class Fleet;
+class DataNode;
+
+
+
+// Information about how a fleet intended for raiding behaves.
+class RaidFleet {
+public:
+	RaidFleet(const Fleet *fleet, double minAttraction, double maxAttraction);
+	static void Load(const DataNode &node, std::vector<RaidFleet> &raidFleets, bool remove, int valueIndex);
```

Comment:
Having a static Load and passing in the container that the new object will become a part of isn't typically how we do this type of thing. I feel like we need a comment explaining why it was done this way.
```suggestion
	// A static loading method for RaidFleet which handles the addition and removal of RaidFleets from the given
	// vector. Implemented this way so that Government and System don't need to duplicate the loading of
	// raid fleets.
	static void Load(const DataNode &node, std::vector<RaidFleet> &raidFleets, bool remove, int valueIndex);
```

Reply:
I mean another class as abstraction isnt how we do it either but this still feels the closest
a comment is def good to have